biojava-structure-gui/src/main/java/org/biojava/nbio/structure/align/gui/DBResultTable.java

```diff
@@ -143,11 +146,11 @@ else if ( str.startsWith("#param:scoring=")){
 							}
 						}
 					} catch (IndexOutOfBoundsException e){
-						System.err.println("Unknown scoring strategy from line: " + str);
+						LOGGER.error("Unknown scoring strategy from line: " + str);
 					} catch (IllegalArgumentException e) {
-						System.err.println("Unknown scoring strategy from line: " + str);
+						LOGGER.error("Unknown scoring strategy from line: " + str);
 					} catch (Exception e) {
-						System.err.println("Unknown parameter can't read parameters from line: " + str);
+						LOGGER.error("Unknown parameter can't read parameters from line: " + str);
 						e.printStackTrace();
```

Comment:
`printStackTrace` prints to stderr. To get the same effect through logging you need `LOGGER.error(e.getMessage(), e)`. Can you include `printStackTrace`s in your script?